README.md

```diff
@@ -39,7 +39,7 @@ You may use a subdirectory inside your DATA_DIR.
 
 First, make sure you actually *have* a DATA_DIR, by creating a folder named 'data' within the main project directory.
 
-e.g TEMPLATE_DIRECTORY=/home/carloscoda/projects/templating/data/templates
+e.g TEMPLATE_DIRECTORY=/home/carloscoda/projects/plato-api/data/templates
```

Comment:
I think that the current repo name is just "plato", so if you check it out directly from github the folder name will also be "plato".

Reply:
you are absolutely right! (for me plato is stored on a local folder named plato-api hence my confusion)